src/langcheck/metrics/config/metric_config.ini

```diff
@@ -0,0 +1,22 @@
+[zh]
+[[semantic_similarity]]
+    # According to the C-MTEB Benchmark
+    # (https://github.com/FlagOpen/FlagEmbedding/tree/master/C_MTEB)
+    # the 3 models of different sizes provided BAAI are the best on the
+    # embedding task
+    # Ref: https://huggingface.co/BAAI/bge-base-zh-v1.5
+    # Using this model, it is hard to find two sentence where cos_sim < 0.25.
+    model_name = BAAI/bge-base-zh-v1.5
+    revision = f03589c
+    loader = sentence-transformers
+[[sentiment]]
+    model_name = IDEA-CCNL/Erlangshen-Roberta-110M-Sentiment
+    loader = huggingface
```

Comment:
thanks, I may choose to change this to hf_text-classifcation to align with the function name in loader. if we always use sentence-transformer for embedding (hope the OOM problem solved), and huggingface for classification, I can update this section to "task = embedding" or "task = classificaiton"

Reply:
If we update the format of the config file though, it could break users' code so I think we should try to not change it as much as possible? E.g. if I (as a user) have code that's like
```
model_manager.set_model_for_metric('zh', 'sentiment', 'some_custom_model_name', loader='huggingface')
```
But then LangCheck gets updated to no longer recognize the loader `huggingface`, it'll be a breaking change for me.

A temporary solution for this might be to first launch the Model Manager _without_ the `set_model_for_metric` method, since that's the only one that could cause issues with compatibility. Then, once we have a stable format for the config that we don't expect to change that often, we could add the `set_model_for_metric` to make it customizable for users. What do you think?

Reply:
yes, change config format frequent make user confused, definitely. Compare with fully remove it, how about make it a private function first, I think we can use set_metric_for_lang in model_manager init phase. The modemanger  set_metric_for_lang one by one when initiated，logically the same.

Reply:
Ah good idea, yes let's make it private first then!